src/client/apps/edit/components/admin/components/article/article_publish_date.tsx

```diff
@@ -49,8 +51,8 @@ export class ArticlePublishDate extends Component {
 
   onUnschedule = () => {
     const { onChange } = this.props
-    let publish_date = null
-    let publish_time = null
+    const publish_date = null
+    const publish_time = null
```

Comment:
Setting `null` to a const feels a little odd. Perhaps these assignments could be inlined?